### PR TITLE
[20.09] marker: init at 2020.04.04

### PR DESCRIPTION
**Pull request by trepetti**
Backporting to 20.09 at user request.

(cherry picked from commit 7559a99131a41d23bc084863ef026e18547ac9df)

###### Motivation for this change

User request to backport to 20.09: https://github.com/NixOS/nixpkgs/pull/100653#issuecomment-735916514

Needs confirmation: https://github.com/NixOS/nixpkgs/pull/100653#issuecomment-739080569

CC: @jonringer 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by trepetti** [comment]
@alexeymuranov in the meantime as we wait for a review, you may enjoy Apostrophe. That's what I've been using recently. The side by side view IMO is nicer than Marker. Only found it after I'd packaged Marker, of course :-)

https://github.com/ApostropheEditor/Apostrophe

**Comment by alexeymuranov** [comment]
@trepetti, thanks, i already have it, but somehow i've got used marker (installed with flatpak, with some issues).

**Review by rmcgibbo — APPROVED** [review]
Builds cleanly and the app works on x86_64-linux.

**Comment by trepetti** [comment]
Thanks for the review, @rmcgibbo. Will close this one, though, since 21.05 is just around the corner which more-or-less obsoletes this PR.

**Comment by jonringer** [comment]
oops, looks like i had a pending review from long, long ago

**Pull request closed without merging**
